ntp-udp/src/socket.rs

```diff
@@ -304,13 +305,14 @@ fn recv(
 
             #[cfg(target_os = "linux")]
             ControlMessage::ReceiveError(_error) => {
-                warn!("unexpected error message on the MSG_ERRQUEUE");
+                tracing::info!("unexpected error message on the MSG_ERRQUEUE");
             }
 
             ControlMessage::Other(msg) => {
-                warn!(
+                tracing::info!(
                     "weird control message {:?} {:?}",
-                    msg.cmsg_level, msg.cmsg_type
+                    msg.cmsg_level,
+                    msg.cmsg_type
```

Comment:
This feels to me like we should either
 - treat it like a bug (hence warn)
 - treat it as non-important operating system behaviour (hence debug  or lower)

The current choice feels weird to me.

Reply:
warn